### ISSUE TITLE
Release innsight 0.4.0

### ISSUE THREAD

**Issue by nkoenen** [bips-hb/innsight#18]
New features:

- [x] Direct usage of gradient-based attribution methods for `torch` models without model conversion 
(✅  Branch `v0.4.0` e0d78c51aea25329f1cc67b37c89353876381f22)
- [x] Add support for `keras3` (successor to `keras` 2.x) 
(✅  Branch `v0.4.0` d5ad83d88fe083886b02497d646b9ab964eb0270)

Bug fixes:

- [x] Fix compatibility issues with legacy `tf-keras` (TensorFlow 2.15 and earlier)
(✅  Branch `v0.4.0` d5ad83d88fe083886b02497d646b9ab964eb0270)

Prepare for release:

* [x] `git pull`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_innsight.html)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)  ✅  Branch `v0.4.0`1a7f450552d22e785cb4a508104147f22392c061
* [ ] `usethis::use_github_links()`
* [ ] `urlchecker::url_check()`
* [ ] `devtools::build_readme()`
* [ ] `devtools::check(remote = TRUE, manual = TRUE)`
* [ ] `devtools::check_win_devel()`
* [ ] `revdepcheck::revdep_check(num_workers = 4)`
* [ ] Update `cran-comments.md`
* [ ] `git push`
* [ ] Draft blog post

Submit to CRAN:

* [ ] `usethis::use_version('minor')`
* [ ] `devtools::submit_cran()`
* [ ] Approve email

Wait for CRAN...

* [ ] Accepted :tada:
* [ ] Finish & publish blog post
* [ ] Add link to blog post in pkgdown news menu
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version(push = TRUE)`
* [ ] Share on social media
